cmd/notation/blob.go

```diff
@@ -0,0 +1,126 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"github.com/notaryproject/notation/cmd/notation/internal/experimental"
+	"github.com/notaryproject/notation/internal/cmd"
+	"github.com/spf13/cobra"
+)
+
+func blobSignCommand(opts *signOpts) *cobra.Command {
+	if opts == nil {
+		opts = &signOpts{
+			inputType: inputTypeRegistry, // remote registry by default
+		}
+	}
+	longMessage := `Sign artifacts
```

Comment:
Can you use the log message as is from `notation blob sign` section in https://github.com/notaryproject/notation/pull/811/files?diff=unified&w=0?